inc/admin/metabox/src/components/MetaFieldsManager.js

```diff
@@ -0,0 +1,70 @@
+/* global metaSidebar */
+import { PanelBody } from "@wordpress/components";
+import { RadioImage } from "./controls/RadioImage"
+import { CheckBox } from "./controls/CheckBox";
+import { Range } from "./controls/Range";
+const { __ } = wp.i18n;
+
+let componentsGroup = [
+	{
+		'title'    : __( 'Layout' ,'neve' ),
+		'controls' : [ 'neve_meta_container', 'neve_meta_sidebar' ]
+	},
+	{
+		'title'    : __( 'Components', 'neve' ),
+		'controls' : [ 'neve_meta_disable_header', 'neve_meta_disable_title', 'neve_meta_disable_featured_image', 'neve_meta_disable_footer' ]
+	},
+	{
+		'title'    : __( 'Content width', 'neve' ),
+		'controls' : [ 'neve_meta_enable_content_width', 'neve_meta_content_width' ]
+	}
+];
+
+let renderControl = ( data, index ) => {
+	if( data.type === 'radio' ){
```

Comment:
`data` can be destructured into `const = {type, id, settings} = data;`. Not a must, but can make the code a bit cleaner.